css_tasks/task4/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Task 4</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <section class = "schedule">
```

Comment:
you don't need spaces before and after  `=` in attributes

Reply:
Done